server/utils/placements/index.test.ts

```diff
@@ -39,6 +39,7 @@ describe('placementUtils', () => {
       const current = typeFactory.current()
       const departed = typeFactory.departed()
       const nonArrival = typeFactory.nonArrival()
+      const cancelled = cas1SpaceBookingFactory.cancelled()
```

Comment:
Likewise, this gets round the fact that the test wouldn't pass on a `Cas1SpaceBookingSummary`

Reply:
This feels like a smell! Maybe the API should return a `isCancelled` property instead? It's a bit more involved, but that's what happens for the `nonArrival` scenario, so it would make sense.

Reply:
True - that would be the right thing. Trouble is, currently the cancelled placements are filtered out of all of the lists so there is no material gain from doing the work, but it might be useful in the future. I'll see if the API guys are up for it.

Reply:
Should be febreezed now.